tests/src/JIT/HardwareIntrinsics/Multiply.cs

```diff
@@ -0,0 +1,181 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Where should I place tests in my PRs - per your discussion with @tannergooding they should go to `x86` directory. Is this still holding?

Reply:
cc @fiigii @tannergooding